### ISSUE TITLE
Python 2.6 support

### ISSUE THREAD

**Issue by alasdairnicol** [Wirecloud/wirecloud#66]
From [this stack overflow question](http://stackoverflow.com/questions/33393848/syntaxerror-compressing-wirecloud).

The docs say that Python 2.6 is supported, but [platform/workspace/utils.py](https://github.com/Wirecloud/wirecloud/blob/271d81206e2fc9e19843c59cd49e45b8d71a8f7b/src/wirecloud/platform/workspace/utils.py#L519) contains dictionary comprehensions, that are only supported by Python 2.7+

**Comment by aarranz** [comment]
Thanks for reporting it :+1:

**Comment by aarranz** [comment]
This is first a bug in the installation guide, because we dropped support for python 2.6 while developing WireCloud 0.8 (e.g our jenkins instance were not passing our tests using python 2.6).

Having said this, we had made some tests for evaluating whether we can reenable python 2.6 support and we have found that the latests versions of some of the dependencies are not working in python 2.6 (e.g. latests versions of the markdown module). So our idea is to remove python 2.6 from the installation documentation and make clear that WireCloud is not compatible with python 2.6.

do anyone need python 2.6 support?
